### DISCUSSION TITLE
I wonder where these data come from? Are they named in the same way as iso3166?

### DISCUSSION THREAD

**Discussion by jabbawockeez** [dr5hn/countries-states-cities-database#191] [Q&A] [answered]

**Comment by dr5hn** [comment, marked as answer]
Hi @jabbawockeez 

This data comes from this source ➡️ https://www.wikidata.org/wiki/Wikidata:Main_Page

ISO3166 are for country_codes you can find that in countries > iso3 column.

Hope this helps.

Thanks
